src/meta/name.rs

```diff
@@ -153,21 +153,21 @@ impl Name {
                 format!(
                     "{}{}",
                     icons.get(self),
-                    self.hyperlink(self.escape(self.file_name(), quote), hyperlink)
+                    self.hyperlink(self.escape(self.file_name(), !quote), hyperlink)
```

Comment:
Let me know if you think `quote` arg should change to literal as well in order to avoid this negation here

Reply:
Yes! it would be great if you can also update this one!

Reply:
Fixed it as well!